_seattle/Washington Gender Alliance Walla Walla

```diff
@@ -13,12 +13,11 @@ email:
 mailing_address:
 contact_form:
 non_english_services: 
-additional_information: For more information and updates request to join the Washington Gender Alliance Facebook group at: <https://www.facebook.com/groups/WashingtonGenderAlliance/>.
+additional_information: "For more information and updates request to join the Washington Gender Alliance Facebook group at: <https://www.facebook.com/groups/WashingtonGenderAlliance/>."
 categories:
   - Support Groups
 supercategories:
   - Mental Health
 features:
-  - confidential
-  - free
----
\ No newline at end of file
+  - Confidential
+  - Free
```

Comment:
Nit: add a newline at the end of this file. (It's generally good style to include a newline at the end of files.)